src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-bucket-list/rgw-bucket-list.component.html

```diff
@@ -42,3 +42,10 @@
   <ng-template #noObjectQuota
                i18n>No Limit</ng-template>
 </ng-template>
+
+<ng-template #deleteTpl>
+  <cd-alert-panel type="info"
+                  i18n>
+    Buckets might still have underlying data depending on your bucket configuration
+  </cd-alert-panel>
+</ng-template>
```

Comment:
This IMO needs to be danger message.

Reply:
I don't think it should be a danger message, we are not able to garantee to the user that the operation is dangerous. We know it might be. But this will depend on theirt configuration, right? And at the time of deletion we don't have the proper information to know if this opertion is dangerous or not.

Eg. Lets say, I, as a user create a bunch of buckets, later on, before managing any data I realize I made I mistake and want to delete one of the recently created (empty) buckets, so I select the delete action and a danger message sudenly appears...What's the danger here?
 I think this could be confusing

Reply:
hmm, for most cases the remove action is dangerous or can cause data loss or something if deleted without proper cleanup/backup. So we can safely assume that the more deserving type of the alert is danger. Let's assume the worst for every scenarios and let the user decide whether this is bad or good. I think it would be okay for UX and won't be confusing. So let's go with danger.

Reply:
Okay, could you provide me a text for this panel? Thanks

Reply:
The current text is okay for me.